shell/platform/tizen/flutter_tizen_engine_test.cc

```diff
@@ -0,0 +1,115 @@
+// Copyright 2020 Samsung Electronics Co., Ltd. All rights reserved.
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <unistd.h>
+#include <chrono>
+#include <thread>
+#include "gtest/gtest.h"
+
+#include "flutter/shell/platform/tizen/flutter_tizen_engine.h"
+
+using namespace std::chrono_literals;
+
+std::string TPK_ROOT_PATH = "/tpkroot";
+std::string LIB_PATH = "/lib";
+std::string RES_PATH = "/res";
```

Comment:
I doubt if these constant definitions are required but if you want to use these constants you should follow the Google naming convention.

Please try to follow the style guide whenever possible when writing the engine code. https://google.github.io/styleguide/cppguide.html

Reply:
It's removed. Thanks for your suggestion!